Client/ZeebeClient.cs

```diff
@@ -135,28 +135,9 @@ public override AsyncUnaryCall<TResponse> AsyncUnaryCall<TRequest, TResponse>(TR
             }
         }
 
-        /// <summary>
-        /// Adds keepAlive options to the channel options.
-        /// </summary>
-        /// <param name="channelOptions">the current existing channel options.</param>
-        private void AddKeepAliveToChannelOptions(List<ChannelOption> channelOptions, TimeSpan? keepAlive)
```

Comment:
Wait we didnt used this anymore? 😃😅

Reply:
I don't know... It's private, so no risk.